src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -123,7 +123,9 @@ export const TransactionsProvider = ({ children }: TransactionsProviderProps): J
                     msg => {
                       const caip10 = `${msg.caip2}:${msg.address}`
                       const state = store.getState()
-                      const accountId = selectAccountIdByAddress(state, caip10)
+                      const accountId = selectAccountIdByAddress(state, {
+                        accountSpecifier: caip10,
```

Comment:
comments, we really need a clear clarification on the delineation between accountSpecifiers and caip10, but that's outside the scope of this PR.

Reply:
I'll create a GitHub conversation for that, that's something we've discussed multiple times with @DaoDev44 and @elmutt

Reply:
https://github.com/shapeshift/web/discussions/1683